### PR TITLE
Default categories

### PR DESCRIPTION
**Pull request by AaronHampshire**
Per Dan's comments, categories are in two columns as clickable buttons. (He also wanted descriptions underneath each category, but I'm not sure how that will actually look and it's not a trivial thing to do.)

The list of optional categories is dynamic, only showing the categories that haven't been chosen. Because categories are deleted with ajax calls, this requires an ajax call to determine which categories are still "available".

If a user enters the same category in the custom field and submits at the same time, there is an error that says the category already exists. I think this is appropriate. 

Messages are also included. If more than one optional categories are added, the number is indicated, rather than listing each category.

### CODE REVIEW

File: RealEstate/apps/core/models.py [hunk 1]
```diff
@@ -31,7 +31,47 @@
...
+    },
+    'category9': {
+        'summary': 'Category9',
+        'description': 'Description for category 9'
+    },
+    'category10': {
+        'summary': 'Category10',
+        'description': 'Description for category 10'
+    }
+    # ....
```
Comment (cdschmitz) [inline RealEstate/apps/core/models.py:74]: Is Dan giving us a real list to replace these?

File: RealEstate/apps/core/static/style.css [hunk 2]
```diff
@@ -133,6 +133,36 @@ button.close{
 	padding-left:10%
 }
 
+.categoryBtn {
+    padding: 5px;
+    margin:10px 0px !important;
+    background-color: #fff;
+    border: 1px solid #ddd;
+    width: 100%;
+}
```
Comment (cdschmitz) [inline RealEstate/apps/core/static/style.css:143]: Should also add an overflow:hidden, the text flows outside of the button when the window is small

**Comment by cdschmitz** [comment]
Works well enough :+1:

File: RealEstate/apps/core/models.py (hunk 1 above)
Comment (AaronHampshire) [inline RealEstate/apps/core/models.py:74]: I'll ask for a list tomorrow.

**Pull request merged**